packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -16,7 +16,7 @@ import {
   FacetValue,
   SearchStatus,
   SearchStatusState,
-} from '@coveo/headless';
+} from '../../../../headless';
```

Comment:
Looks like this caused a change in `components.d.ts`. We typically import from `@coveo/headless`, so I think you should revert this line.

Reply:
Yes!